apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/AbstractR4ResourceProvider.java

```diff
@@ -50,15 +52,18 @@
 import org.hl7.fhir.r4.model.Bundle;
 import org.hl7.fhir.r4.model.Claim;
 import org.hl7.fhir.r4.model.ClaimResponse;
+import org.hl7.fhir.r4.model.DomainResource;
 import org.hl7.fhir.r4.model.IdType;
 import org.hl7.fhir.r4.model.Resource;
+import org.springframework.stereotype.Component;
 
 /**
  * Allows for generic processing of resource using common logic. Claims and ClaimResponses have the
  * exact same logic for looking up, transforming, and returning data.
  *
  * @param <T> The specific fhir resource the concrete provider will serve.
  */
+@Component
 public abstract class AbstractR4ResourceProvider<T extends IBaseResource>
```

Comment:
Abstract classes do not need to be components when using constructor based injection.  Concrete instances are the components.

Reply:
Thanks, removed it!